lib/rides/pages/ride_detail_page.dart

```diff
@@ -78,6 +80,9 @@ class _RideDetailPageState extends State<RideDetailPage> {
       final int id = _ride?.id ?? widget.id!;
       final Map<String, dynamic> data =
           await SupabaseManager.supabaseClient.from('rides').select(_rideQuery).eq('id', id).single();
+      final Map<String, dynamic> dataChat =
```

Comment:
In the _rideQuery, you can put

```
    chats: chats!chats_ride_id_fkey(
      *,
      messages(*)
    )
```

That would save us from this monkey-patching

Reply:
I see you added that part, but you did not remove this line

Reply:
changed it the problem is that i need dataChat to use single

Reply:
That problem could also be avoided if you switched the association around